documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -406,6 +406,192 @@ To obtain a list of caches available in a {brandname} cluster, invoke a `GET` re
 include::rest_examples/get_v2_list_caches.adoc[]
 ----
 
+[[rest_v2_cache_xsite]]
+=== Cross site replication
+
+Use the REST API to monitor and control Cross Site (x-site) replication on your {brandname} cluster.
+See link:../xsite/xsite.html[Cross Site replication] for more details about this feature.
```

Comment:
@oraNod we are trying to link the x-site page here (since it has more details about the operation below).
is the link correct?

Reply:
@pruivo yes, that link is correct to get to the xsite docs but should be community only. I've asked Gustavo about adding conditional text so it doesn't appear in product docs.

Reply:
@pruivo as agreed, I'll do a separate doc PR after this is merged.